PhysicsTools/NanoAOD/python/custom_jme_cff.py

```diff
@@ -272,7 +272,9 @@ def AddPileUpJetIDVars(proc, jetName="", jetSrc="", jetTableName="", jetTaskName
       vertexes  = "offlineSlimmedPrimaryVertices",
       inputIsCorrected = True,
       applyJec  = False,
-      usePuppi = True if "PUPPI" in jetName.upper() else False
+      usePuppi = True if "PUPPI" in jetName.upper() else False,
+      srcConstituentWeights = "packedPFCandidatespuppi" if "PUPPI" in jetName.upper() else "",
+      useBugFix = True
```

Comment:
Is this default expected to be `True` (and `srcConstituentWeights` empty)?
Where is this custom_jme_cff.py used?

Reply:
`custom_jme_cff.py` is used for JMENano production. JetMET is planning a JMENano campaign for 2022 with the bug-fixed version of the PileUp Id input variables included, so `useBugFix` is set to `True` here to ensure this. `srcConstituentWeights` is default to be empty for CHS jets, which is what `pileupJetIdCalculator` is setup for by default.